src/molecules/AccountIndicator/index.js

```diff
@@ -209,4 +209,9 @@ AccountIndicator.propTypes = {
   onMenuLinkClick: PropTypes.func
 };
 
+AccountIndicator.defaultProps = {
+  dashboardPath: '/users/dashboard',
```

Comment:
I'm sure you're aware, but we'll just need to make sure to override this default for renter's.

Reply:
good ol renters! :+1: